apps/src/main.rs

```diff
@@ -52,30 +46,20 @@ struct Args {
     /// URL of the Ethereum RPC endpoint.
     #[clap(short, long, env)]
     rpc_url: Url,
-    /// Private key used to interact with the EvenNumber contract.
-    #[clap(short, long, env)]
-    private_key: PrivateKeySigner,
-    /// Submit the request offchain via the provided order stream service url.
-    #[clap(short, long, requires = "order_stream_url")]
-    offchain: bool,
```

Comment:
Let's keep the `offchain` flag. It shows that it can be done both ways, and how to switch easily.